test/Table_Tests/src/Database/Helpers/Fake_Test_Connection.enso

```diff
@@ -1,33 +1,47 @@
 from Standard.Base import all
 
+from Standard.Database import SQL_Query, Raw_SQL, Table_Name
 import Standard.Table.Data.Table as Materialized_Table
 import Standard.Database.Data.Table as Database_Table
 from Standard.Database.Data.SQL import Statement, SQL_Type
+import Standard.Database.Internal.IR
 
 type Fake_Test_Connection
     # type Fake_Test_Connection_Data (tables : Map Text (Vector [Text, SQL_Type])) (dialect : Text)
     Fake_Test_Connection_Data tables dialect
 
-    ## PRIVATE
-    access_table : Text -> Database_Table
-    access_table self name =
-        columns = self.tables.get name
-        Database_Table.make_table self name columns
+    ## UNSTABLE
+
+       Query the database, returning an object that can be used to retrieve create queries for that table.
+
+       Arguments:
+       - name: name of the table to access
+    query : Text | SQL_Query -> Text -> Database_Table
+    query self query _="" = case query of
```

Comment:
```suggestion
    query self query alias="" =
        _ = alias
```

I don't think it's safe replacing the arg with `_` here.

If someone calls `connection.query "foo" alias="bar"` - with the alias is provided as a named parameter, then if the Fake connection is used, it will not have an argument called `alias` and such an invocation will fail.

This is an issue in Enso where we need a method conform to a more generic interface, including supporting all named parameters, while not all parameters may be used in some variants of the method.

While this is used only for testing maybe we can ignore this altogether as long as it works. But I wanted to highlight the issue in general